clients/src/main/java/org/apache/kafka/clients/consumer/internals/OffsetCommitCallbackInvoker.java

```diff
@@ -33,7 +32,6 @@
  */
 public class OffsetCommitCallbackInvoker {
     private final ConsumerInterceptors<?, ?> interceptors;
-    private boolean hasFencedException = false;
```

Comment:
Since we are setting it now from `AsyncKafkaConsumer`, it seemed cleaner to move it from `OffsetCommitCallbackInvoker` to `AsyncKafkaConsumer`.